### PR TITLE
Extracting out title component from confirm-transaction-base

### PR DESCRIPTION
**Pull request by jpuri**
Extracting out title component from confirm-transaction-base

Ref: #17246

### CODE REVIEW

File: ui/components/app/confirm-title/confirm-title.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    >
+      <UserPreferencedCurrencyDisplay
+        ethLogoHeight={24}
+        hideLabel={!isContractInteraction}
+        showCurrencySuffix={isContractInteraction}
+        showEthLogo
+        type={PRIMARY}
+        value={hexTransactionAmount}
+      />
+    </Text>
```
Comment (digiwand) [inline ui/components/app/confirm-title/confirm-title.js:56]: nice cleanup! 

does this code also support removing the use of `titleComponent` passed from `ui/pages/confirm-transaction-base/confirm-transaction-base.component.js` to ConfirmPageContainer?

File: ui/pages/confirm-transaction-base/confirm-transaction-base.component.js [hunk 2]
```diff
@@ -837,22 +838,11 @@ export default class ConfirmTransactionBase extends Component {
   renderTitleComponent() {
```
Comment (digiwand) [inline ui/pages/confirm-transaction-base/confirm-transaction-base.component.js:838]: Nice cleanup! 

could we remove the use of `renderTitleComponent` and the `titleComponent` props? It looks like the new ConfirmTitle component you extracted supports this unless I'm missing something

Comment (jpuri) [reply ui/pages/confirm-transaction-base/confirm-transaction-base.component.js:838]: I still left the function to avoid initialising the component inline.

**Review by digiwand — DISMISSED** [review]
LGTM!

**Review by digiwand — DISMISSED** [review]

**Review by digiwand — DISMISSED** [review]

**Review by segun — DISMISSED** [review]

**Review by digiwand — DISMISSED** [review]

**Review by digiwand — DISMISSED** [review]

**Review by segun — APPROVED** [review]

**Review by digiwand — APPROVED** [review]

**Pull request merged**